helm/nginx-ingress-controller-app/Chart.yaml

```diff
@@ -1,5 +1,5 @@
 apiVersion: v1
-appVersion: v0.27.1
+appVersion: v0.28.0
```

Comment:
This was missed in 1.3.0; I've added a note in values.yaml for the tag, when changing the tag to change also appVersion